diffusion/auxiliary.py

```diff
@@ -69,4 +69,4 @@ def loss_at_step_t(self, x0, t, loss_type="l1", noise=None):
         # TODO: how should a_loss be weighted?
         loss = p_loss + 1e-3*a_loss
 
-        return loss
+        return loss, {}
```

Comment:
Why do we need this? Wouldn't this break the signature of the class, i.e. break the contract that the subclass has with its superclass?